openstudiocore/src/model/AirflowNetworkDistributionLinkage.hpp

```diff
@@ -53,7 +55,7 @@ class MODEL_API AirflowNetworkDistributionLinkage : public AirflowNetworkLinkage
   /** @name Constructors and Destructors */
   //@{
 
-  explicit AirflowNetworkDistributionLinkage(const Model& model);
+  AirflowNetworkDistributionLinkage(const Model& model, const AirflowNetworkNode &node1, const AirflowNetworkNode &node2, const AirflowNetworkComponent &component);
```

Comment:
@jasondegraw typically we don't require the user to pass in model here, you can call nodel1.model() to get this in the ctor